src/components/structures/RightPanel.tsx

```diff
@@ -73,7 +73,7 @@ interface IState {
 export default class RightPanel extends React.Component<IProps, IState> {
     static contextType = MatrixClientContext;
 
-    private readonly delayedUpdate: RateLimitedFunc;
+    private readonly delayedUpdate: DebouncedFunc<() => void>;
```

Comment:
this could be

```
private delayedUpdate = throttle((): void => {
    this.forceUpdate();
}, 500, { leading: true, trailing: true });
```

which is more akin other arrow methods

Reply:
and like we seem to do in other places as exemplified below